### PR TITLE
Camera

### PR DESCRIPTION
**Pull request by PrabhmeharSingh**
Merge Camera , gallery image delete functionality.
Merge Conflicts present, please fix before merging.
Clash with Fahrin branch intentintegrator

### CODE REVIEW

File: app/src/main/AndroidManifest.xml
Comment (Fahrin0802) [inline app/src/main/AndroidManifest.xml:1]: what is the difference between this and just requiring camera permissions?

Comment (PrabhmeharSingh) [reply app/src/main/AndroidManifest.xml:1]: If u mean uses feature then that is required to start an action.Camera intent

File: app/src/main/java/com/example/myapplication/EditItemFragment.java
[the remark this answers is not in the document]
Comment (PrabhmeharSingh) [reply app/src/main/java/com/example/myapplication/EditItemFragment.java:1]: Is the branch compiling?

File: app/src/main/java/com/example/myapplication/AddItemFragment.java
Comment (Fahrin0802) [inline app/src/main/java/com/example/myapplication/AddItemFragment.java:1]: solved. And existing UI tests pass. We should be good

**Review by Fahrin0802 — APPROVED** [review]
Awesome job on this!

**Pull request merged**